app.py

```diff
@@ -0,0 +1,21 @@
+from flask import (
+    jsonify,
+    render_template,
+    Flask,
+)
```

Comment:
Is that the import style you've adopted? I don't think it's that common and it may not scale really well, but I see the motivation.

Reply:
It's for when you want clean diffs. With the trailing comma on the end, when you add an import, you get a one line diff.

Reply:
Like I said, I see the motivation :stuck_out_tongue: 
I was just asking about your usage of it.

Reply:
Sorry, I was saying that more for other's benefit. :)